### PR TITLE
refactor: changed the default behavior of the `kurtosis feedback` command, now it opens the GitHub choose new issue page

### PR DESCRIPTION
**Pull request by leoporoli**
## Description:
Changed the default behavior of the `kurtosis feedback` command, now it opens the GitHub choose new issue page. The help text was edited also.

## Is this change user-facing?
YES

## References (if applicable):
Fixes https://github.com/kurtosis-tech/kurtosis-private/issues/1195 

The new help text:
image

### CODE REVIEW

File: docs/docs/cli-reference/feedback.md [hunk 1]
```diff
@@ -4,14 +4,11 @@ sidebar_label: feedback
...
+To quickly get in touch with us, simply run:
 ```bash
-kurtosis feedback
+kurtosis feedback 
 ```
-which will return:
-- The link to our [GitHub](https://github.com/kurtosis-tech/kurtosis/issues/new/choose) for filing bug reports and feature requests as GitHub Issues. 
-- Our [feedback email address](mailto:feedback@kurtosistech.com) for sending us suggestions, comments, or questions about Kurtosis via email.
-- A [Calendly link](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding) to schedule a dedicated 1:1 onboarding session with us to help you get started.
+which will open the [Kurtosis GitHub choose new issue page](https://github.com/kurtosis-tech/kurtosis/issues/new/choose), where you can select the kind of feedback you want and then describe what you want.
```
Comment (leeederek) [inline docs/docs/cli-reference/feedback.md]:
```suggestion
which will open the [Kurtosis GitHub create new issue page](https://github.com/kurtosis-tech/kurtosis/issues/new/choose), where you can file a bug report, submit a feature request, or let us know about any docs issues you may have come across. A link to our [Discord server](https://discord.gg/yUGgeE8s) will be made available there too.
```

**Review by leeederek — APPROVED** [review]
I cannot comment on the Go code but the docs look good to go!

File: cli/cli/commands/feedback/feedback.go [hunk 2]
```diff
@@ -24,7 +21,15 @@ const (
 		"See below for the many ways you can get in touch with us.\n\n" +
 		"TIP: You can quickly type and send us feedback directly from the CLI. For example, " +
 		"`kurtosis feedback \"I enjoy the enclave naming theme\"` will open the Kurtosis GitHub " +
-		"\"Choose New Issue\" page with the description pre-filled with \"I enjoy the enclave naming theme\"."
+		"\"Choose New Issue\" page with the description pre-filled with \"I enjoy the enclave naming theme\".\n\n" +
+		"This command can be used to deliver feedback to the Kurtosis team!\n\n" +
+		"* Pass in your feedback as an argument using double quotations (e.g. kurtosis feedback \"my feedback\"), and press enter; this will open our GitHub Issues templates, pre-filled with your feedback/arg.\n" +
+		"* Pass in the --email flag to open a draft email, pre-filled with your feedback/arg, to send to feedback@kurtosistech.com.\n" +
+		"* Pass in the --calendly flag to open our Calendly link to schedule a 1:1 session with us for feedback and questions you may have!\n\n" +
```
Comment (h4ck3rk3y) [inline cli/cli/commands/feedback/feedback.go:28]: All of this sounds good! we are duplicating some information given by the flags themselves I suppose. Is that okay?

File: docs/docs/cli-reference/feedback.md [hunk 3]
```diff
@@ -4,14 +4,11 @@ sidebar_label: feedback
 slug: /feedback
 ---
 
-To quickly see all the ways you can get in touch with us, simply run:
+To quickly get in touch with us, simply run:
 ```bash
-kurtosis feedback
+kurtosis feedback
```
Comment (h4ck3rk3y) [inline docs/docs/cli-reference/feedback.md]: extra space here

**Review by h4ck3rk3y — APPROVED** [review]

File: cli/cli/commands/feedback/feedback.go [hunk 4]
```diff
@@ -24,7 +21,15 @@ const (
 		"See below for the many ways you can get in touch with us.\n\n" +
 		"TIP: You can quickly type and send us feedback directly from the CLI. For example, " +
 		"`kurtosis feedback \"I enjoy the enclave naming theme\"` will open the Kurtosis GitHub " +
-		"\"Choose New Issue\" page with the description pre-filled with \"I enjoy the enclave naming theme\"."
+		"\"Choose New Issue\" page with the description pre-filled with \"I enjoy the enclave naming theme\".\n\n" +
```
Comment (h4ck3rk3y) [inline cli/cli/commands/feedback/feedback.go]: s/choose/create

if you accept @leeederek 's suggestion from docs

**Review by h4ck3rk3y — APPROVED** [review]

File: cli/cli/commands/feedback/feedback.go (hunk 2 above)
Comment (leoporoli) [reply cli/cli/commands/feedback/feedback.go:28]: yes, because there are small differences between both places

**Pull request merged**